pootle/templates/staticpages/_body.html

```diff
@@ -9,7 +9,14 @@
   {% endblocktrans %}
   </div>
   {% endif %}
-  <h1 class="title">{{ page.title }}</h1>
+  <h1 class="title">
+    {{ page.title }}
+    {% if user.is_superuser %}
+    <span class="edit-link">
+      [<a href="{% url 'pootle-staticpages-edit' pk=page.id page_type='announcements' %}">edit</a>]
```

Comment:
Forget about this. Noticed that `page.get_edit_url` should provide us the URL without need to use the `url` templatetag.